src/content/translations/uk/security/index.md

```diff
@@ -0,0 +1,289 @@
+
+Деякі правила, про які потрібно пам’ятати:
+
+- Будьте обережними з тими, хто звертається до вас із пропозицією заробити на вашій криптовалюті.
+- Вивчіть інформацію про стейкінг, пули ліквідності й інші способи інвестування криптовалюти.
+- Якщо такі пропозиції й бувають справжніми, то дуже рідко. Якби вони були поширеними, про них би говорили всі й ви б теж про них чули.
+
+[Жертва втратила 200 тисяч доларів через шахрайську схему, пов’язану з пулом для майнингу](https://www.reddit.com/r/CoinBase/comments/r0qe0e/scam_or_possible_incredible_payout/)
+
+### Шахрайство, пов’язане з токенами Eth2 {#eth2-token-scam}
+
+Оскільки у 2022 році відбудеться [об’єднання](/upgrades/merge/), шахраї користуються плутаниною навколо терміна «Eth2» і намагаються змусити користувачів обміняти ETH на токен «ETH2». Жодних «ETH2» або будь-яких інших нових токенів не буде представлено після об’єднання. Токени ETH, якими ви володієте сьогодні, залишаться такими самими ETH після об’єднання. Вам не потрібно обмінювати ETH через об’єднання.
+
+Шахраї можуть видавати себе за працівників служби підтримки, повідомляючи вам, що якщо ви внесете токен ETH, то отримаєте назад «ETH2». [Офіційної служби підтримки Ethereum](/community/support/), як і нового токена, не існує. Нікому не повідомляйте кодову фразу свого гаманця.
+
+_Примітка. Існують похідні токени / тікери, які можуть позначати частку ETH у стейкінгу (як-от rETH від Rocket Pool, stETH від Lido, ETH2 від Coinbase), але на них не потрібно «переходити»._
+
+### Шахрайство з роздачею {#airdrop-scams}
+
+Шахрайство з роздачею передбачає, що зловмисний проєкт надсилає актив (NFT, токен) у ваш гаманець і спрямовує вас на шахрайський сайт, на якому можна його отримати. За спроби отримати актив вам буде запропоновано ввійти в систему за допомогою гаманця Ethereum і «схвалити» транзакцію. Ця транзакція скомпрометує ваш обліковий запис, надсилаючи шахраєві ваші відкриті й приватні ключі. Альтернативною формою цієї схеми може бути підтвердження транзакції, яка надсилає кошти на рахунок шахрая.
+
+[Докладніше про шахрайства з роздачею](https://www.youtube.com/watch?v=LLL_nQp1lGk)
+
+<Divider />
+
+## Додаткові ресурси {#further-reading}
+
+### Веббезпека {#reading-web-security}
+
+- [Ось чому вам не слід використовувати повідомлення для двофакторної автентифікації](https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin) — _The Verge_
+- [До 3 мільйонів пристроїв заражено вірусами від розширень Chrome і Edge](https://arstechnica.com/information-technology/2020/12/up-to-3-million-devices-infected-by-malware-laced-chrome-and-edge-add-ons/) — _Ден Ґудін_
+- [Як створити надійний пароль, який ви не забудете](https://www.avg.com/en/signal/how-to-create-a-strong-password-that-you-wont-forget) — _AVG_
+- [Що таке ключ безпеки?](https://help.coinbase.com/en/coinbase/getting-started/verify-my-account/security-keys-faq) — _Coinbase_</ul>
```

Comment:
```suggestion
- [Що таке ключ безпеки?](https://help.coinbase.com/en/coinbase/getting-started/verify-my-account/security-keys-faq) — _Coinbase_
```

Reply:
Another dodgy `<ul>` tag... I think this was caused by the first item in the list as there is no evidence in Crowdin of there being an issue with this item